R/comparePlot.R

```diff
@@ -141,133 +233,121 @@ comparePlot <- function(compareDF,
             var.annot <- NULL
             colorBy   <- NULL
             sizeBy    <- NULL
             colors    <- symbolFill[2]
-            sizes     <- as.numeric(symbolSize[2])
+            sizes     <- symbolSize[2]
         }
-
-        CompPlot <- canvasXpress::canvasXpress(data                    = cx.data,
-                                               varAnnot                = var.annot,
-                                               decorations             = decorations,
-                                               graphType               = "Scatter2D",
-                                               colorBy                 = colorBy,
-                                               colors                  = colors,
-                                               legendPosition          = legendPosition,
-                                               scatterAxesEqual        = TRUE,
-                                               showDecorations         = TRUE,
-                                               sizeBy                  = sizeBy,
-                                               sizes                   = sizes,
-                                               sizeByShowLegend        = FALSE,
-                                               title                   = title,
-                                               xAxisTitle              = xlab,
-                                               yAxisTitle              = ylab,
-                                               citation                = footnote,
-                                               citationFontSize        = footnoteSize,
-                                               citationColor           = footnoteColor)
+        canvasXpress::canvasXpress(data             = cx.data,
+                                   varAnnot         = var.annot,
+                                   decorations      = decorations,
+                                   graphType        = "Scatter2D",
+                                   colorBy          = colorBy,
+                                   colors           = colors,
+                                   legendPosition   = legendPosition,
+                                   scatterAxesEqual = TRUE,
+                                   showDecorations  = TRUE,
+                                   sizeBy           = sizeBy,
+                                   sizes            = sizes,
+                                   sizeByShowLegend = FALSE,
+                                   title            = title,
+                                   xAxisTitle       = xlab,
+                                   yAxisTitle       = ylab,
+                                   citation         = footnote,
+                                   citationFontSize = footnoteSize,
+                                   citationColor    = footnoteColor
+        )
     } else {
+        if (any(is.null(symbolShape),
```

Comment:
You are supposed to have defaults for canvasXpress as well. I am only seeing defaults for ggplot

Reply:
both symbolColor && symbolShape are not used for canvasXpress, do we have a way to use them as they are used in ggplot?

Reply:
You can use parameter shapes and colors for symbolColor and symbolShape

Reply:
* for colors I used for both based on your previous comment
* for shapes, that parameter is working differently in ggplot than cx plot and based on different values, do you have a suggestion for that? thank you

Reply:
Yes I have created lookup functions for Shapes in ggplot and cxplot. Refer to the ggplotMDS function which you are testing currently. In internals.R there are functions .get_valid_shapes_ggplot, .is_valid_shape_ggplot, .get_valid_shapes_cxplot, .is_valid_shape_cxplot,